UnityProject/Assets/Scripts/Messages/Client/RequestConveyorBuildMessage.cs

```diff
@@ -17,16 +17,27 @@ public struct NetMessage : NetworkMessage
 			//index of the entry in the ConstructionList.
 			public byte EntryIndex;
 			public ConveyorBelt.ConveyorDirection Direction;
+			public bool Sandboxing;
 		}
 
 		public override void Process(NetMessage msg)
 		{
 			var playerScript = SentByPlayer.Script;
+			if (msg.Sandboxing)
```

Comment:
Permissions need to be checked here, as the client can set `Sandboxing` when they want.

Reply:
I'll add an admin check in here